api/v1alpha1/dspipeline_types.go

```diff
@@ -122,25 +143,28 @@ type MlPipelineUI struct {
 type Database struct {
 	*MariaDB    `json:"mariaDB,omitempty"`
 	*ExternalDB `json:"externalDB,omitempty"`
+	// Default: false
 	// +kubebuilder:default:=false
 	// +kubebuilder:validation:Optional
 	DisableHealthCheck bool `json:"disableHealthCheck"`
 }
 
 type MariaDB struct {
+	// Default: true
 	// +kubebuilder:default:=true
 	// +kubebuilder:validation:Optional
 	Deploy bool   `json:"deploy"`
```

Comment:
"Enable DS Pipelines Operator management of MariaDB. Setting Deploy to false disables operator reconciliation."